entries/e/eki-net.com.json

```
@@ -0,0 +1,16 @@
+{
+  "Ekinet": {
+    "domain": "eki-net.com",
+    "url": "https://www.eki-net.com/personal/top/index",
```

Better let the website do the redirection by itself.
```suggestion
    "url": "https://www.eki-net.com/",
```